pinot-api/src/main/java/com/linkedin/pinot/client/ExternalViewReader.java

```diff
@@ -114,4 +120,27 @@ public ExternalViewReader(ZkClient zkClient) {
     return tableToBrokersMap;
   }
 
+  private static byte[] unpackZnodeIfNecessary(byte[] znodeContents) {
```

Comment:
This is a temporary workaround right, we should be using Helix API to get the EV - that would have handled compression automatically.